Formula/comby.rb

```diff
@@ -14,6 +14,7 @@ class Comby < Formula
   depends_on "gmp" => :build
   depends_on "ocaml" => :build
   depends_on "opam" => :build
+  depends_on "autoconf"
```

Comment:
Is this also a runtime dependency, or is it just build-only?

Reply:
you're right, thanks for catching. `pkg-config` should be as well.